platforms/google/tectonic.tf

```diff
@@ -92,6 +95,18 @@ module "tectonic" {
   experimental      = "${var.tectonic_experimental}"
   master_count      = "${var.tectonic_master_count}"
   stats_url         = "${var.tectonic_stats_url}"
+
+  image_re = "${var.tectonic_image_re}"
+}
+
+module "flannel-vxlan" {
```

Comment:
i think we could also add the calico policy plugin here: https://github.com/coreos/tectonic-installer/blob/master/platforms/aws/tectonic.tf#L119

Reply:
Thanks! I will add calico and approach more networking stuff (https://github.com/enxebre/tectonic-installer/issues/13) on a separate PR, I wanted to focus on etcd here and just have some basic SDN to get etcd working

Reply:
definitely! we can ship this first and do calico as a follow-up. it is alpha anyways.